values_test.go

```diff
@@ -572,6 +577,13 @@ func TestNumberValue_ToBigEndianBytes(t *testing.T) {
 			NewWord64(9223372036854775808):  {128, 0, 0, 0, 0, 0, 0, 0},
 			NewWord64(18446744073709551615): {255, 255, 255, 255, 255, 255, 255, 255},
 		},
+		"Word128": {
+			NewWord128(0):   {0},
+			NewWord128(42):  {42},
+			NewWord128(127): {127},
+			NewWord128(128): {128},
+			NewWord128(200): {200},
+		},
```

Comment:
Same as above, maybe add some test for larger values, and improve the tests for `UInt128` at the same time

Reply:
Done